Bottom: da2f09831f39cac1620a98c833c02d3c356a4c7b
Top:    ef3f5cc8efb9bce833be0fa3c18013a2c49e878a
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-18 18:35:13 +1300

Refresh of extended-to-stream

---

diff --git a/doc/for-framework-folk.rst b/doc/for-framework-folk.rst
index 492e9b7..84c92e3 100644
--- a/doc/for-framework-folk.rst
+++ b/doc/for-framework-folk.rst
@@ -161,6 +161,15 @@ is observed. Aborting multiple workers in a distributed environment requires
 hooking whatever signalling mechanism the distributed environment has up to
 a ``TestControl`` in each worker process.
 
+ExtendedToStreamDecorator
+-------------------------
+
+This is a hybrid object that combines both the ``Extended`` and ``Stream``
+``TestResult`` APIs into one class, but only emits ``StreamResult`` events.
+This is useful when a ``StreamResult`` stream is desired, but you cannot
+be sure that the tests which will run have been updated to the ``StreamResult``
+API.
+
 ThreadsafeStreamResult
 ----------------------
